projects/igniteui-angular/src/lib/grids/grid-toolbar.component.html

```diff
@@ -39,7 +39,7 @@
                 <span></span>
             </div>
         </button>
-        <igx-drop-down #columnPinningDropdown>
+        <igx-drop-down #columnPinningDropdown [style.min-width.px]="180">
```

Comment:
I think this should be done with a class, or inside the toolbar theme. @simeonoff

Reply:
@kdinev should this be a global setting for the drop-down menu? Like the drop-down should not go bellow 180 px?

Reply:
@simeonoff I'm not sure. Is there a guideline for this in material?

Reply:
@kdinev Yes, there is a min-width of 128px for the drop down menus.

Reply:
@simeonoff Makes sense. Implement it globally then!